frontend/src/components/modals/AddIncome.js

```diff
@@ -0,0 +1,200 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+import Box from '@mui/material/Box';
+import TextField from '@mui/material/TextField';
+import Autocomplete from '@mui/material/Autocomplete';
+import Switch from '@mui/material/Switch';
+
+export default function AddIncome() {
+  const [type, setType] = useState('');
+  const [billingDate, setBillingDate] = useState('');
+  const [endBillingDate, setEndBillingDate] = useState('');
+  const [repeatInterval, setRepeatInterval] = useState('');
+  const [amount, setAmount] = useState('');
+  const [description, setDescription] = useState('');
+  const [recipient, setRecipient] = useState('');
+  const [accountName, setAccountName] = useState('');
+
+  const types = ['Wydatek', 'Przychód'];
+  const [accounts, setAccounts] = useState([]);
+
+  const [checkedRepeat, setCheckedRepeat] = useState(false);
+  const [checkedNotifications, setCheckedNotifications] = useState(false);
+
+  const handleRepeatChange = (event) => {
+    setCheckedRepeat(event.target.checked);
+  };
+
+  const handleNotificationsChange = (event) => {
+    setCheckedNotifications(event.target.checked);
+  };
+
+  useEffect(() => {
+    fetch('http://localhost:8080/AccountsList')
+      .then((res) => res.json())
+      .then((json) => {
+        setAccounts(json.map((obj) => obj.accountName));
+      });
+  }, []);
+
+  function handleSubmit(event) {
+    event.preventDefault();
+    axios({
+      method: 'post',
+      url: 'http://localhost:8080/incomes-expenses',
+      params: {
+        billingDate: billingDate,
+        endBillingDate: endBillingDate,
+        repeatInterval: repeatInterval ? repeatInterval : 0,
```

Comment:
@lstachon @Kiszkolot @zofiagrodecka
Poproszę potwierdzenie od kogoś z backendu, czy jeżeli wydatek się nie powtarza, to należy wysłać `repeatInterval` jako zero?

Reply:
Tak, potwierdzam, ma być 0